app/src/components/market/common/eth_balance/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TokenEthereum } from '../../../../util/types'
+import { EtherIcon } from '../../../common/icons/currencies'
+
+const Wrapper = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 13px;
+  height: 40px;
+  border-radius: 8px;
```

Comment:
@liaojikunwork maybe we should use this value from themes for button in case in the future we change that...

Reply:
@Mi-Lan , this component is only for eth_balance, and that kind of cleanup is not for this PR.
Need to add more fields on index.js for eth_balance component.

Reply:
@liaojikunwork i mean there is borderRadius for cards dunno if we should create another just for eth_balance component...
![Screenshot 2020-11-23 at 17 06 06](https://user-images.githubusercontent.com/33226956/99985113-2dbf4000-2dae-11eb-91ff-8a608969c451.png)

Reply:
@Mi-Lan , cards and buttons are not for such components.